packages/client/src/utils/constants.ts

```diff
@@ -92,4 +96,9 @@ const METHODS = {
   DELETE: 'DELETE',
 }
 
-export { CONTENT_RU, PATHNAMES, API, METHODS }
+const TYPE = {
+  SUCCESS: 'success',
+  ERROR: 'error',
+}
```

Comment:
Может enum использовать?
```suggestion
enum TYPE {
  SUCCESS = 'success',
  ERROR = 'error'
}
```